### PR TITLE
Fix opening first channel on server start

### LINKED ISSUE #1547: When running master for the first time, there is no default channel
When running from master, on first launch before there is any active channel, The Lounge just displays no channel, which appears broken on a mobile browser, and may not be clear on a desktop/larger screen. Once a channel is manually selected, it works fine going forward as it remembers what was selected. astorije mentioned they saw the same issue on first launch.

### PR DESCRIPTION
**Pull request by xPaw**
Fixes #1547. Auto reconnect PR broke this behaviour, so I'm bringing it back.

### CODE REVIEW

File: client/js/socket-events/init.js [hunk 1]
```diff
@@ -50,25 +50,30 @@ socket.on("init", function(data) {
...
+	let target = $();
 
 	// Open last active channel
 	if (clientActive > 0) {
 		target = sidebar.find("[data-id='" + clientActive + "']");
 	}
 
 	// Open window provided in location.hash
-	if (!target && window.location.hash) {
+	if (!target.length > 0 && window.location.hash) {
```
Comment (astorije) [inline client/js/socket-events/init.js]: === 0
Same below

Comment (xPaw) [reply client/js/socket-events/init.js]: oops

Comment (astorije) [reply client/js/socket-events/init.js]: Instead of !...> 0 I mean

Comment (xPaw) [reply client/js/socket-events/init.js]: Fixed it.

**Pull request merged**